stps/sig-network/ip-request-stp.md

```diff
@@ -0,0 +1,216 @@
+# Openshift-virtualization-tests Test plan
+
+## **Request a specific IP for a VM in UDN through a third party integration - Quality Engineering Plan**
+
+### **Metadata & Tracking**
+
+| Field                   | Details                                                                       |
+|:----------------------- |:----------------------------------------------------------------------------- |
+| **Enhancement(s)**      | [As close as possible to VEP](https://access.redhat.com/solutions/7133388)    |
```

Comment:
a link to the original enhancement - https://github.com/openshift/enhancements/pull/1793 - should be included